sensor/kubernetes/listener/resources/convert.go

```diff
@@ -336,13 +337,23 @@ func (w *deploymentWrap) populateImageMetadata(localImages set.StringSet, pods .
 
 			image := w.Deployment.Containers[i].Image
 
-			// If there already is an image ID for the image then that implies that the name of the image was fully qualified
-			// with an image digest. e.g. quay.io/stackrox-io/main@sha256:xyz
-			// If the ID already exists, populate NotPullable and IsClusterLocal.
+			var runtimeImageName *storage.ImageName
+			if features.UnqualifiedSearchRegistries.Enabled() && c.ImageID != "" {
```

Comment:
does `image.GetId() == c.ImageID` or is this like the configured name vs the runtime name? Can you provide an example for each? I thought the ID was the digest, but it may be the fully qualified name?